### DISCUSSION TITLE
Where is AdGuardHome directory?

### DISCUSSION THREAD

**Discussion by johnaaronrose** [AdguardTeam/AdGuardHome#6495] [Q&A]
After installing & setting up AdGuard on a Raspberry Pi model 4 under Raspbian, there is no AdGuardHome directory nor AdGuard app in /opt,  /bin, /sbin, /usr/bin, /usr/sbin, ~. I don't understand why AdGuard or similar is not shown in Task Manager. But AdGuard seems to be running according to its Dashboard!
After installing & setting up AdGuard, there is no AdGuardHome directory nor AdGuard app in /opt,  /bin, /sbin, /usr/bin, /usr/sbin, ~. I don't understand why AdGuard or similar is not shown in Task Manager. But AdGuard seems to be running according to its Dashboard!
![AdGuardDashboard](https://github.com/AdguardTeam/AdGuardHome/assets/397379/5aba1941-b1d6-44bb-908c-90438533f142)
[https://app.box.com/s/t1chilwtsy3nulnq512x2ce7txshj23r](url)

**Comment by Debcharon** [comment]
`find / -name AdGuardHome`

[Default directory](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#unix-linux-macos-bsd): `/opt/AdGuardHome`

```plain-test
/opt/AdGuardHome
/opt/AdGuardHome/agh-backup/AdGuardHome
/opt/AdGuardHome/AdGuardHome
```

**Reply by johnaaronrose** [reply]
As I said in my question, there is no AdGuardHome or similarly named directory in /opt/.
